pages/linux/gsettings.md

```diff
@@ -0,0 +1,32 @@
+# gsettings
+
+> Query and modify dconf settings with schema validation.
+> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/using_the_desktop_environment_in_rhel_8/configuring-gnome-at-low-level_using-the-desktop-environment-in-rhel-8#using-gsettings-command_configuring-gnome-at-low-level>.
+
+- Display all (non-relocatable) schemas, keys, and values:
+
+`gsettings list-recursively`
+
+- Display all keys and values (default if not set) from one schema:
```

Comment:
What does the `default` mean here? Is this behaviour default or does it use a default schema?

Reply:
The default value for that key.

AFAICT, GSettings uses a dconf database backend that only stores changes from the default values specified in the schema definition.  Schema definitions are found in `/usr/share/glib-2.0/schemas`.

So:

```
$ gsettings reset org.gnome.settings-daemon.plugins.xsettings hinting
$ gsettings get org.gnome.settings-daemon.plugins.xsettings hinting
'slight'
$ dconf read /org/gnome/settings-daemon/plugins/xsettings/hinting
$ # (no output)
```

But if we explicitly set that key, even to one that matches the default...

```
$ gsettings set org.gnome.settings-daemon.plugins.xsettings hinting slight
$ gsettings get org.gnome.settings-daemon.plugins.xsettings hinting
'slight'
$ dconf read /org/gnome/settings-daemon/plugins/xsettings/hinting
'slight'
```

It is stored in the dconf database.

(And incidentally, we have triggered [a very old cairo bug](https://gitlab.freedesktop.org/cairo/cairo/-/issues/23).)